cmd/bridge/main.go

```diff
@@ -108,6 +108,8 @@ func main() {
 
 	fLoadTestFactor := fs.Int("load-test-factor", 0, "DEV ONLY. The factor used to multiply k8s API list responses for load testing purposes.")
 
+	fHelmChartRepoUrl := fs.String("helm-chart-repo-url", "https://redhat-developer.github.com/redhat-helm-charts", "Helm chart repository URL")
```

Comment:
We'll need a flag to the CA file as well. When set to an in-cluster service, it should use the service CA cert

Reply:
@spadgett 

FYI:

After our meeting last day, we simplified the design to not need an in-cluster service at all. 

Console UI --> Console backend ( proxy ) --> public charts repo ( https://redhat-developer.github.io/redhat-helm-charts/index.yaml ) 

We've taken out the intermediate in-cluster service which was talking to Github ( no new pods anymore!)

As for 'disconnected' coverage, I've outline the plan for the disconnected/configurable/customer-specific helm charts repo scenario here https://github.com/openshift/enhancements/pull/175/files/cb34545ea0b13a23de7faaa940976a6be156ba59#diff-c191c073bb34ddf6a2325f2b95abc87dR32 . 

In 4.4, we'll ensure 'disconnected' is possible. In 4.5, we'll enhance the experience.

Reply:
```suggestion
	fHelmChartRepoURL := fs.String("helm-chart-repo-url", "https://redhat-developer.github.com/redhat-helm-charts", "Helm chart repository URL")
```

Reply:
@sbose78 Will users on disconnected clusters see an error in the UI before the additional configuration is done? We're defaulting to an external chart repo here. Do we need to give a way to unset the value?

Reply:
The plan is to have additional PR that would expose the url on [ConsoleSpec](https://github.com/openshift/api/blob/master/config/v1/types_console.go#L26) so that console operator can react on changes and update `config.yaml` consumed by the backend.